src/tiles/grid/item-tile.ts

```diff
@@ -223,19 +218,20 @@ export class ItemTile extends LitElement {
 
       .item-creator {
         color: #2c2c2c;
-        display: -webkit-box;
         font-size: 1.4rem;
         height: 3rem;
-        margin: 0px;
         overflow: hidden;
-        padding: 0.5rem;
         text-align: center;
         text-overflow: ellipsis;
         -webkit-line-clamp: 2;
         -webkit-box-orient: vertical;
+        display: flex;
+        justify-content: center;
+        align-items: flex-end;
+        padding-top: 1rem;
       }
 
-      #text-by {
+      #by {
```

Comment:
Does this `#by` element exist? I didn't notice it above.

Reply:
good catch... i just removed it